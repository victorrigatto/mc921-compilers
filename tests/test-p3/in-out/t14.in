int main(char i, char j)
{
  int i, j;
  return i+j;
}
