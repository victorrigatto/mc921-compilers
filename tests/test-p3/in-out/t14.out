SemanticError: Name i is already defined in this scope @ 3:7
